cfme/tests/services/test_myservice.py

```diff
@@ -45,7 +46,9 @@ def myservice(setup_provider, provider, catalog_item, request):
     Metadata:
         test_flag: provision
     """
-    vm_name = catalog_item.provisioning_data["vm_name"] + "_0001"
+    vm_name = version.pick({
```

Comment:
We need to keep this as is. This is not us making up a name, this is a name that ManageIQ assigns to the VM.